starter-code/src/functions-and-arrays.js

```diff
@@ -60,8 +155,24 @@ var wordsFind = [
   'disobedience'
 ];
 
+doesWordExist = function(array, word) {
+
+let newWords;
+if(array.length === 0) {
+  return false;
+}
+for (let i = 0; i < array.length; i++) {
+  if (word === array[i]) {
+ return newWords = true;
+  } else {
+     newWords = false;
```

Comment:
for this function you can just return true or false in your if else statement. There is no need to have a variable hold the boolean that you will return;